docs/release_notes/v0.10.0.md

```diff
@@ -0,0 +1,457 @@
+  
+# Dapr 0.10.0
+
+We're happy to announce the release of Dapr 0.10.0!
+
```

Comment:
nit, `v0.10` or v0.10.0`

Reply:
also consistency throughout the release notes doc on the format of version

Reply:
Changed